src/hostapi/wasapi/pa_win_wasapi.c

```diff
@@ -2934,14 +2940,16 @@ static HRESULT GetAlternativeSampleFormatExclusive(IAudioClient *client, double
 }
 
 // ------------------------------------------------------------------------------------------
-static PaError GetClosestFormat(IAudioClient *client, double sampleRate, const PaStreamParameters *_params,
-    AUDCLNT_SHAREMODE shareMode, WAVEFORMATEXTENSIBLE_UNION *outWavexU, BOOL output)
+static PaError GetClosestFormat(IAudioClient *client, const PaWasapiDeviceInfo *deviceInfo,
+    const PaStreamParameters *_params, double sampleRate, AUDCLNT_SHAREMODE shareMode,
+    WAVEFORMATEXTENSIBLE_UNION *outWavexU, BOOL output)
 {
     PaWasapiStreamInfo *streamInfo   = (PaWasapiStreamInfo *)_params->hostApiSpecificStreamInfo;
     WAVEFORMATEX *sharedClosestMatch = NULL;
     HRESULT hr                       = !S_OK;
     PaStreamParameters params        = (*_params);
     const BOOL explicitFormat        = (streamInfo != NULL) && ((streamInfo->flags & paWinWasapiExplicitSampleFormat) == paWinWasapiExplicitSampleFormat);
+    const BOOL sharedMode            = (shareMode == AUDCLNT_SHAREMODE_SHARED);
```

Comment:
maybe nicer
```suggestion
    const BOOL isSharedMode            = (shareMode == AUDCLNT_SHAREMODE_SHARED);
```

Reply:
I was also thinking about it but decided to obey the style of `explicitFormat` variable. Changed to ­`is` version and also updated `explicitFormat` to play nicely together.